README.md

```diff
@@ -1,127 +1,3 @@
-
-    1. **Locate the Pre-SBE repository**: This is where you'll be making changes.
-
-    1. **Check `eng/Versions.props`**: Look for a version property for your component. If it doesn't exist, you'll need to add it.
-
-    1. **Check `eng/Version.Details.xml`**: Similarly, look for an entry for your component. If it doesn't exist, add it.
-
-    1.  **Add a descriptive comment**: When adding an entry to `Version.Details.xml`, include a comment above the entry that describes what it is.
-
-    1. **Examples**: For reference, you can check these examples of [`eng/Versions.props`](https://github.com/dotnet/arcade/pull/14698/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R81) and [`eng/Version.Details.xml`](https://github.com/dotnet/arcade/pull/14698/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3R8).
-
-1. **Update the component**: 
-
-    1. **Wait for changes to propagate**: If you added a Version.Details.xml dependency and corresponding property in the Versions.Props for the component that you are updating, then you need to wait for these changes to flow to the [VMR](https://github.com/dotnet/dotnet) before you can update the component. If there is already a Version.Details.xml dependency and corresponding property in the Versions.Props, then there is no need to wait and you can move on to the next step immediately.
-
-    1. **Update the component**: Once the changes have propagated, you can update the component as usual. For guidance, follow the steps in [`Updating an External Component to a Newer Version`](#updating-an-external-component-to-a-newer-version).
-
-## Patches
-
-1. When creating/updating patches, it is desirable to backport the changes whenever feasible as this reduces
-the maintenance burden when [updating a component to a newer version](#updating-an-external-component-to-a-newer-version).
-
-1. Steps to create new patches:
-
-    1. Make changes in the submodule.
-
-    1. Commit changes in the submodule.
-
-    1. From the root directory of the submodule, run [extract-patches.sh](src/repos/patches/extract-patches.sh)/[extract-patches.ps1](src/repos/patches/extract-patches.ps1).
-       The script will prepare a patch based on the base sha of the submodule and the latest committed changes. The patch
-       will be added to patches/<component>/*.patch
-
-1. To apply a patch, or multiple patches, use `git am` while inside the submodule directory
-For example, to apply *all* `humanizer` patches:
-
-```sh
-# cd src/humanizer
-# git am "../../patches/humanizer/*"
-```
-## Found an Issue?
-
-Source build related issues are tracked in the [source build repo](https://github.com/dotnet/source-build/).
-
-## License
-
-This repo is licensed under the [MIT](LICENSE.txt) license.
+Going forward, the .NET team is using [dotnet/source-build-reference-packages](https://github.com/dotnet/source-build-reference-packages) to develop the code formerly in this repository.
```

Comment:
Considering adding that pre-10.0 versions are still available in this repo in their respective release branches.